sdk/python/examples/quickstart/Quickstart.ipynb

```diff
@@ -124,13 +123,12 @@
     "# Initialise client\n",
     "fs = Client(core_url=FEAST_CORE_URL, verbose=True)\n",
     "\n",
-    "serving_ds=Datastore(id='REDIS1')\n",
-    "warehouse_ds=Datastore(id='NOOP')\n",
+    "serving_ds=Datastore(id='SERVING')\n",
```

Comment:
You could remove it actually, it will use the default store

Reply:
removed on 9b33b80